Library/Homebrew/dev-cmd/test-bot.rb

```diff
@@ -842,12 +904,15 @@ def test_ci_upload(tap)
     ENV["HUDSON_SERVER_COOKIE"] = nil
     ENV["JENKINS_SERVER_COOKIE"] = nil
     ENV["HUDSON_COOKIE"] = nil
+    ENV["COVERALLS_REPO_TOKEN"] = nil
 
     ARGV << "--verbose"
 
-    if jenkins
+    bottles = Dir["*.bottle*.*"]
+    if bottles.empty? && jenkins
```

Comment:
@sjackman, this is the "hairy" change I was referring to with test-bot. It's essentially a very mangled merge-conflict resolution of https://github.com/Homebrew/brew/commit/2ec5423356e51dde343b1fa69e3e7c07c3aa4a8e

I think it's probably fine from the looks of it, but just wanted to point it out since it's the only change I felt nervous about.